frontend/src/app/components/enterprise-banner/enterprise-banner-bootstrap.component.ts

```diff
@@ -1,8 +1,10 @@
-import {Component, OnInit, ElementRef} from "@angular/core";
-import {DynamicBootstrapper} from "core-app/globals/dynamic-bootstrapper";
+import {Component, ElementRef, OnInit} from "@angular/core";
 import {I18nService} from "core-app/modules/common/i18n/i18n.service";
 
+export const enterpriseBannerSelector = 'enterprise-banner-bootstrap';
+
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/3b1eae5a64159f8a044579a25870c772186dbd3f/frontend/src/app/components/enterprise-banner/enterprise-banner-bootstrap.component.ts#L5"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>